<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	color: #FFF;
}

#centralwidget {
	background: #000;
}

QComboBox, QPushButton, QLineEdit {
	padding: 0 15px;
	border-radius: 15px;
	background: #555;
}
QComboBox {
	text-align: center;
}

QListView {
	padding: 15px;
	border-radius: 15px;
	background: #555;
}
QProgressBar  {
	border-radius: 15px;
	background-color: #555;
	text-align: center;
}
QProgressBar::chunk {
	border-radius: 15px;
	background-color: #999;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Url</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineUrl">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Src</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineSrc">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browse">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="comboBoxes" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="QChange">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <item>
             <property name="text">
              <string>One</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Playlist</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="TChange">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>10</pointsize>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>Video</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Audio</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="convert">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Ð¡onvert</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="info" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListView" name="listView">
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="cursor" stdset="0">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="autoScroll">
             <bool>true</bool>
            </property>
            <property name="autoScrollMargin">
             <number>16</number>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="isWrapping" stdset="0">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="textVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
